Framework/API/CMakeLists.txt

```diff
@@ -121,6 +121,7 @@ set(SRC_FILES
     src/ParameterReference.cpp
     src/ParameterTie.cpp
     src/PeakFunctionIntegrator.cpp
+    src/PreviewDeclarations.cpp
```

Comment:
I believe this file can be safely suppressed, as the methods inside are not called anywhere, and the widget works as intended without.